django/db/migrations/recorder.py

```diff
@@ -84,3 +84,31 @@ def flush(self):
         Deletes all migration records. Useful if you're testing migrations.
         """
         self.migration_qs.all().delete()
+
+
+class NullMigrationRecorder(object):
```

Comment:
Don't we already have something like that somewhere in the migration tests? If so, mind adapting those occurrences?